packages/nextjs/app/builds/page.tsx

```diff
@@ -0,0 +1,16 @@
+import { AllBuilds } from "./_components/AllBuilds";
+import { BuildCategory, BuildType } from "~~/services/database/config/types";
+import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
+
+export const metadata = getMetadata({
+  title: "All Builds",
+  description: "View all the builds by the Speed Run Ethereum community",
+});
+
+export default async function AllBuildsPage({
+  searchParams,
+}: {
+  searchParams: { category?: BuildCategory; type?: BuildType };
+}) {
```

Comment:
Note: I am using search params from the server side page because the build was failing when I had this whole page under `"use client"`. It was related to https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout